quake-mode@repsac-by.github.com/quakemodeapp.js

```diff
@@ -179,56 +177,55 @@ var QuakeModeApp = class {
 	show() {
 		const { actor, focusout } = this;
 
-		if ( this.state !== state.RUNNING )
-			return;
-
-		if ( Tweener.isTweening(actor) )
+		if ( this.state !== state.RUNNING || this.isAnimating )
 			return;
 
-		const quake = this;
-
-		Tweener.addTween(actor, {
+		actor.ease({
 			translation_y: 0,
 			clip_y: 0,
-			time: this.ainmation_time,
-			onStart() {
-				this.get_parent().set_child_above_sibling(actor, null);
-				this.set_clip(0, this.height, this.width, this.height);
-				actor.translation_y = - actor.height,
-				this.show();
+			duration: this.animation_time,
+			mode: Clutter.AnimationMode.EASE_OUT_QUAD,
+			onStart: () => {
+				this.isAnimating = true;
+				actor.get_parent().set_child_above_sibling(actor, null);
+				actor.set_clip(0, this.height, this.width, this.height);
+				actor.translation_y = - actor.height;
+				actor.show();
 			},
```

Comment:
I didn't find an implementation of onStart in gnome-shell, in my opinion it just doesn't work.

Reply:
You're right! I somehow missed that.

Still, there's an animation, the window's just not visible.